inc/macro_utils/macro_utils.h

```diff
@@ -238,20 +236,6 @@ typedef struct ENUM_VALUE_AND_STRING_TAG
 }ENUM_VALUE_AND_STRING;
```

Comment:
also not used from what I can tell

Reply:
Also removed.

---
In reply to: [636327914](https://github.com/Azure/macro-utils-c/pull/75#discussion_r636327914) [](ancestors = 636327914)